<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowDialog</class>
 <widget class="QDialog" name="BorrowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;等线 Light&quot;;</string>
   </property>
   <property name="title">
    <string>借阅/预约</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>540</width>
      <height>300</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_condition">
     <property name="text">
      <string>借阅情况</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_condition">
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>280</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_intro">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;等线 Light&quot;;</string>
     </property>
     <property name="text">
      <string>简介</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 10pt &quot;等线 Light&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;等线 Light&quot;;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_title">
       <property name="text">
        <string>标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_author">
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_publisher">
       <property name="text">
        <string>出版社</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_amount">
       <property name="text">
        <string>借阅数量</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
